src/Layout/Banner.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const BannerSection = styled.div`
+  position: fixed;
+  top: 3.75rem;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 1.8rem;
+  min-width: 100vw;
+  background-color: #2e2e2e;
+  z-index: 10;
+`;
+
+const BannerText = styled.h5`
+  font-family: 'Domine', serif;
+  font-size: 0.825rem;
+  font-weight: 400;
+  text-align: center;
+  color: white;
+  letter-spacing: 1px;
+`;
+
+const Button = styled.button`
+  margin-left: 1.5rem;
+  border: 2px solid #e6c564;
+  background-color: #e6c564;
+  cursor: pointer;
+  &:active {
+    transform: scale(0.9);
+    transition: 0.5s ease-out linear;
+  }
+`;
+
+const Banner = () => {
+  const [closeBanner, setCloseBanner] = useState(true);
+
+  const handleCloseBanner = () => {
+    setCloseBanner(!closeBanner);
+  };
+
+  return (
+    <>
+      {closeBanner && (
+        <BannerSection>
+          <BannerText>Free shipping on all orders $25+</BannerText>
+          <Button type='button' onClick={handleCloseBanner}>
```

Comment:
The [priority list](https://docs.google.com/spreadsheets/d/1dwWKhiQhGnyFRbJns1lnl16Bmi-x7ZNFfUZU1lmJh6I/edit#gid=0) we came up with states that we don't need to be able to close this, did Caitlin or Sam change course on this by chance? Quoteth the spreadsheet:

`Fixed at top, scroll out of view, no need to be able to close, editable via shopify`

Reply:
ok, I finally got your Node version to work. So now we are both on 14.15.1. I removed the legacy Node script I added and the now the project successfully builds. Woo! That was an adventure.

Reply:
I removed the Close button moved the Banner to the top of the page so I am now waiting for your review. Thanks D!

Reply:
This looks great!